taskcat/testing/_cfn_test.py

```diff
@@ -197,10 +198,14 @@ def report(
         cfn_logs = _CfnLogTools()
         cfn_logs.createcfnlogs(self.test_definition, report_path)
         ReportBuilder(
-            self.test_definition, report_path / "index.html"
+            self.test_definition, report_path / "index.html", get_installed_version()
         ).generate_report()
 
 
+def get_installed_version():
+    return importlib.metadata.version(__package__ or __name__)
```

Comment:
I took this from `_cli.py`. I had issues when using that version directly and when attempting to pull it out into `_common_utils.py`.

I might give this part one more attempt and then ask for help.